### ISSUE TITLE
"v-text-anchor" property is broken

### ISSUE THREAD

**Issue by ashtonsix** [emotion-js/emotion#716]
- `emotion` version: 9.2.3
- `react` version: 16.3.2

Minimal Example:

```javascript
import { css } from 'emotion'

const app = document.getElementById('root')
const myStyle = css`
  color: rebeccapurple;
  v-text-anchor: middle;
`

app.classList.add(myStyle)
```

The injected stylesheet looks like this:

```html
<style data-emotion>
  v-text-anchor:middle;.css-1e4y5fg{color:rebeccapurple;}
</style>
```

Without `color` (just `v-text-anchor`) the injected stylesheet looks like this:

```html
<style data-emotion>
  v-text-anchor:middle;}
</style>
```

I would expect the injected stylesheet to look like this instead:

```html
<style data-emotion>
  .css-1e4y5fg{v-text-anchor:middle;}
</style>
```

Other properties that match `/^[a-z]{1}-/` also seem to break.

[![Edit 1o9op8moyq](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/1o9op8moyq)

<details>
<summary>Real-World Example (spoiler):</summary>
There's a few intermediate build steps, but this generates VML for displaying rounded buttons in older versions of Outlook.
<pre><code>return (
  &lt;center>
    &lt;if condition="mso">
      &lt;v$roundrect
        xmlns$v="urn:schemas-microsoft-com:vml"
        xmlns$w="urn:schemas-microsoft-com:office:word"
        href={link}
        className="emailButton"
        css={`
          width: ${width};
          height: ${height};
          v-text-anchor: middle;
        `}
        fillcolor={background}
        arcsize={arcsize}
        stroke="f"
      >
        &lt;w$anchorlock />
        &lt;center
          css={`
            color: ${color};
            font-size: ${fontSize};
          `}
        >
          {children}
        &lt;/center>
      &lt;/v$roundrect>
    &lt;/if>
    &lt;else condition="mso">
      &lt;a
        href={link}
        className="emailButton"
        css={`
          display: block;
          width: ${width}px;
          line-height: ${height}px;
          -webkit-text-size-adjust: none;
          background-color: ${background};
          border-radius: ${borderRadius}px;
          font-size: ${fontSize}px;
          color: ${color};
        `}
      >
        {children}
      &lt;/a>
    &lt;/else>
  &lt;/center>
)</code></pre>
</details>

**Comment by Andarist** [comment]
VML is predating Internet Explorer 10. In general we do not want to support such old environments. Could u use inline styles for this property?

**Comment by thysultan** [comment]
Strange that with would happen for `v-`, testing this on [stylis.js.org](https://stylis.js.org) reveals that it is a bug with stylis.

**Comment by ashtonsix** [comment]
@Andarist yeah, but then I can't tell my colleagues migrating styles from older html templates to react is simple as copy/pasting.

@thysultan thanks for finding! going to report upstream & create a PR to upgrade our Stylis version later. turns out only `/^v-/` breaks, other letters are fine

**Comment by Andarist** [comment]
> @Andarist yeah, but then I can't tell my colleagues migrating styles from older email templates to react is simple as copy/pasting. welcome to the wonderful world of email clients btw.

Yeah, that's certainly a bummer. Hope the issue will get resolved. 

> @thysultan thanks for finding! going to report upstream & create a PR to upgrade our Stylis version later.

You won't quite be able to upgrade stylis, because `emotion` has its own copy of it (it's not regular dependency). When the new version gets released you should create a stylis version bump request here.

**Comment by Ailrun** [comment]
@ashtonsix Here is the `stylis` of `emotion`
https://github.com/emotion-js/next/tree/master/packages/stylis

**Comment by ashtonsix** [comment]
@Andarist Hey, stylis 3.5.1 was just released! Can we upgrade?

**Comment by Andarist** [comment]
https://github.com/emotion-js/next/pull/15
